openslides_backend/action/mixins/import_mixins.py

```diff
@@ -147,21 +269,41 @@ def create_action_result_element(
             "headers": self.headers,
             "rows": self.rows,
             "statistics": self.statistics,
-            "state": self.state,
+            "state": self.import_state,
         }
 
+    def add_payload_index_to_action_data(self, action_data: ActionData) -> ActionData:
+        for payload_index, entry in enumerate(action_data):
+            entry["payload_index"] = payload_index
+        return action_data
+
     def validate_instance(self, instance: Dict[str, Any]) -> None:
         # filter extra, not needed fields before validate and parse some fields
         property_to_type = {
-            header["property"]: header["type"] for header in self.headers
+            header["property"]: (
+                header["type"],
+                header.get("is_object"),
+                header.get("is_list", False),
+            )
+            for header in self.headers
         }
         for entry in list(instance.get("data", [])):
             for field in dict(entry):
                 if field not in property_to_type:
                     del entry[field]
                 else:
-                    type_ = property_to_type[field]
-                    if type_ == "integer":
+                    type_, is_object, is_list = property_to_type[field]
```

Comment:
`is_object` is not used if I see it correctly

Reply:
it is used in the ActionException to descripe an unknown type